tools/build.sh

```diff
@@ -1,5 +1,21 @@
 #!/bin/bash
 
+# replace string $1 by environment variable $2 in file $3
+replace_config_variable() {
+    VARNAME="$2"
+
+    VALUE=$(ECHO "${!VARNAME}")
+
+    if [ "$VALUE" != "" ]; then
+        sed -i -e "s/$1/$VALUE/g" $3
+    fi
+}
+
+# ease configuration of allowed origin by adding quotes
+if [ "$KEYGUARD_ALLOWED_ORIGIN" != "" ]; then
+    KEYGUARD_ALLOWED_ORIGIN="'$KEYGUARD_ALLOWED_ORIGIN'"
```

Comment:
I think you changed the variables names when you were writing it, but you have to decide on either `CONFIG_ALLOWED_ORIGIN` or `KEYGUARD_ALLOWED_ORIGIN`.